packages/material-ui/src/Chip/Chip.js

```diff
@@ -42,6 +41,29 @@ export const styles = theme => {
         opacity: 0.5,
         pointerEvents: 'none',
       },
+      '& $avatar': {
```

Comment:
I'm curious whether or not you deem this a breaking change because strictly speaking specificity is increased 😅

Reply:
The outlined variant already improves the specificity to style the avatar.

Reply:
True - but let's say someone was just specifying a single `avatar` class name, they'd now need to also do `& $avatar`. Maybe not a big concern - I was more just curious to hear your perspective on breaking/non-breaking changes so we can adopt the same principle at my workplace.

Reply:
@NMinhNguyen If somebody has overridden the class, he would likely face #16374, as we do.